Doc/library/email.utils.rst

```diff
@@ -87,7 +92,7 @@ of the new API.
    This method returns a list of 2-tuples of the form returned by ``parseaddr()``.
    *fieldvalues* is a sequence of header field values as might be returned by
    :meth:`Message.get_all <email.message.Message.get_all>`.  Here's a simple
-   example that gets all the recipients of a message::
```

Comment:
@tdwyer Is this right? Isn’t the next paragraph still a literal block? Why have you removed this colon?